applications/luci-app-openvpn/luasrc/model/cbi/openvpn.lua

```diff
@@ -26,7 +26,7 @@ uci:foreach( "openvpn_recipes", "openvpn_recipe",
 )
 
 function s.getPID(section) -- Universal function which returns valid pid # or nil
-	local pid = sys.exec("%s | grep -w '[o]penvpn(%s)'" % { psstring, section })
+	local pid = sys.exec("%s | grep -w '[o]penvpn(%s)' | awk '{print $1}'" % { psstring, section })
 	if pid and #pid > 0 then
 		return tonumber(pid:match("^%d+"))
```

Comment:
I suggest changing the pattern to just `%d+`, without the beginning of line char `^`